shutdown_test.go

```diff
@@ -87,6 +91,43 @@ func TestShutdown(t *testing.T) {
 		assert.NotNil(t, <-done1, "done channel 1 did not receive signal")
 		assert.NotNil(t, <-done2, "done channel 2 did not receive signal")
 	})
+
+	t.Run("with exit code", func(t *testing.T) {
+		t.Parallel()
+		var s fx.Shutdowner
+		app := fxtest.New(
+			t,
+			fx.Populate(&s),
+		)
```

Comment:
I realize this PR didn't add this pattern, but I realize all of these tests share this boilerplate. Maybe worth cleaning this up and converting to a test table?

Reply:
We couldn't generalize the t.Parallel call without abstracting a whole case runner around populate. 

A new fxtest function would be possible, although it doesn't really reduce boilerplate too much, something like: 

`func NewPopulated(t testingT, target interface{}) fx.App { 
   return fxtest.New(t, fx.Populate(target))
}`